apps/wallet/src/ui/app/staking/validators/index.tsx

```diff
@@ -0,0 +1,148 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { is, SuiObject } from '@mysten/sui.js';
+import { useState, useMemo } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { getName, STATE_OBJECT } from '../usePendingDelegation';
+import { SelectValidatorCard } from './SelectValidatorCard';
+import { ValidatorsCard } from './ValidatorsCard';
+import {
+    activeDelegationIDsSelector,
+    totalActiveStakedSelector,
+} from '_app/staking/selectors';
+import Alert from '_components/alert';
+import { SuiIcons } from '_components/icon';
+import Loading from '_components/loading';
+import Overlay from '_components/overlay';
+import { useAppSelector, useObjectsState, useGetObject } from '_hooks';
+
+import type { ValidatorState } from '../ValidatorDataTypes';
+
+export type ValidatorsProp = {
+    name: string;
+    apy: number | string;
+    logo: string | null;
+    address: string;
+    pendingDelegationAmount: bigint;
+};
+
+export function Validators() {
+    const [showModal, setShowModal] = useState(true);
+
+    const navigate = useNavigate();
+    const close = () => {
+        navigate('/');
+    };
+
+    const { loading, error } = useObjectsState();
+    const activeDelegationIDs = useAppSelector(activeDelegationIDsSelector);
+    const totalStaked = useAppSelector(totalActiveStakedSelector);
+    const accountAddress = useAppSelector(({ account }) => account.address);
+    const { data, isLoading } = useGetObject(STATE_OBJECT);
+
+    const validatorsData =
```

Comment:
I actually think this probably would be a better fit for living in the `ValidatorsCard` itself vs having it fetched here then passed down. The only thing that it's actually used for here is conditional rendering of `ValidatorsCard` vs `SelectValidatorCard`, but you could even do that in the `ValidatorCard` itself.

Reply:
The reason I did it here is the page title, which takes the validator name and Icon.